packages/aragon-cli/src/commands/deploy.js

```diff
@@ -36,6 +36,7 @@ exports.builder = yargs => {
 }
 
 exports.task = async ({
+  module,
```

Comment:
```suggestion
  module,
  environment,
```

(see my other comment)

Reply:
I agree, but it seems like only the selected environment is included in `module.envionments` by the middleware. See [here](https://github.com/aragon/aragon-cli/blob/master/packages/aragon-cli/src/middleware/environment.js#L113). Even if that's the case though I agree being explicit here is better.

Reply:
Oh ok, makes sense

Reply:
How about adding the selected environment as `module.env = env` in the environments middleware?